src/ducks/categories/CategoriesPage.jsx

```diff
@@ -36,7 +35,7 @@ class CategoriesPage extends Component {
 
   onWithIncomeToggle = checked => {
     const { settingsCollection, dispatch } = this.props
-    const settings = getSettings(settingsCollection)
+    const settings = get(settingsCollection, 'data[0]')
```

Comment:
Why don't you use `getSettingsFromState` in this component, like you did in `Balance` ?